packages/shared/src/utils/contract/contract.ts

```diff
@@ -154,3 +156,17 @@ export const getContractUpdateConfig = ({ contract_update, limit_order }: TGetCo
 export const shouldShowExpiration = (symbol: string) => /^cry/.test(symbol);
 
 export const shouldShowCancellation = (symbol: string) => !/^(cry|CRASH|BOOM|stpRNG|WLD|JD)/.test(symbol);
+
+export const getContractSubtype = (type: string) => {
+    const contract_type = type.toUpperCase();
+    switch (contract_type) {
+        case 'VANILLALONGCALL':
+        case 'VANILLALONGPUT':
+            return contract_type[11] + type.toLowerCase().slice(12);
+        case 'TURBOSLONG':
+        case 'TURBOSSHORT':
+            return contract_type[6] + type.toLowerCase().slice(7);
+        default:
+            return '';
+    }
+};
```

Comment:
```suggestion
export const getContractSubtype = (type: 'VANILLALONGCALL' | 'VANILLALONGPUT' | 'TURBOSLONG' | 'TURBOSSHORT') => {
    const normalizedType = type.replace(/(VANILLALONG|TURBOS)/, '').toLowerCase();
    const capitalizeType = capitalizeFirstLetter(normalizedType);

    return capitalizeType;
};
```

Reply:
@farzin-deriv we have to leave type as string because this function will receive any contract types without limitations. If the type is vanilla or turbos, we want to get a subtype. But if any other type is received, we want to returned an empty string. However, @kate-deriv has applied capitalizeFirstLetter() in this function. Thank you for pointing it out👍🏻